custom/icds_reports/models/views.py

```diff
@@ -585,6 +585,10 @@ class AggAwcMonthly(models.Model):
     )
     infantometer = models.IntegerField(blank=True, null=True)
     stadiometer = models.IntegerField(blank=True, null=True)
+    valid_visits = models.ImageField(blank=True, null=True,
+                                     help_text="num of valid awc visits")
```

Comment:
nitpick: better These visits are home visits done by aww so calling them awc visits can be misleading. you could call these home visits.

Reply:
done